analyzers/tests/SonarAnalyzer.UnitTest/TestCases/Utilities/MetricsAnalyzer/AllMetrics.cs

```diff
@@ -15,7 +15,15 @@ public int Go(bool condition)
             {
                 return 42;
             }
+
+            var y = string.Empty;
+            (y, var z) = ("a", 'x');
+
             throw new NotImplementedException();
         }
     }
+
```

Comment:
Could you please add a simple struct, I see that the class count in the tests is now 3 which is a bit confusing. As the record struct is technically not a class.

Reply:
sure.